programs/lib/startup.c

```diff
@@ -12,9 +12,10 @@
 #include <flexpret_io.h>
 #include <flexpret_lock.h>
 #include <flexpret_thread.h>
+#include <flexpret_config.h>
 #include "tinyalloc/tinyalloc.h"
 
-#define DSPM_LIMIT          ((void*)0x20040000) // 0x40000 = 256K
+#define DSPM_LIMIT          ((void*) DSPM_END) // 0x40000 = 256K
```

Comment:
Minor comment. Maybe remove the comment, or change it into something more informative?